upi/vsphere/variables.tf

```diff
@@ -131,3 +132,68 @@ variable "compute_ips" {
   type    = "list"
   default = []
 }
+
+//////NSX-T ///////
+variable "nsx_manager" {
+  type        = "string"
+  description = "This is the NSX manager for the environment."
+}
+
+variable "nsx_user" {
+  type        = "string"
+  description = "NSX manager user for the environment."
+}
+
+variable "nsx_password" {
+  type        = "string"
+  description = "NSX manager password"
+}
+
+variable "ip_block_cidr" {
+  type = "string"
+}
+
+variable "ip_pool_start" {
+  type = "string"
+}
+
+variable "ip_pool_end" {
```

Comment:
are these for the external ip pool? If so, prepend all of them with "external"
some of those are optional (like start and end) make sure this is made clear in the descriptin

Reply:
these are for the IP pool defined for the DHCP, that will hold the cluster VMs